src/test/java/core/basesyntax/service/reader/ReaderFileImplTest.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax.service.reader;
+
+import core.basesyntax.model.FruitTransaction;
+import java.io.File;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ReaderFileImplTest {
+    private static final String INPUT_FILE = "src/test/resources/input.csv";
+    private static final String OUTPUT_FILE = "src/main/resources/output.csv";
+    private static final String NEGATIVE_FRUIT = "src/main/resources/negativeFruit.csv";
+    private ReaderFile readerFile;
+    private File file;
+    private FruitTransaction fruitTransaction;
+
+    @BeforeEach
+    void setUp() {
+        readerFile = new ReaderFileImpl();
+        file = new File(INPUT_FILE);
+        fruitTransaction = new FruitTransaction();
+    }
```

Comment:
Please initialize services once for whole test class unless it's necessary to reset context between tests.
Recheck all test classes